template_feed/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Company.WebApplication1.csproj

```diff
@@ -45,4 +50,9 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="1.1.0" Condition="'$(IndividualAuth)' == 'True'" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(Framework)' == 'netcoreapp1.1' AND '$(IndividualAuth)' == 'True'">
+    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.0.0-msbuild3-final" />
+    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.0-msbuild3-final" />
+    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="1.0.0-msbuild3-final" />
+  </ItemGroup>
```

Comment:
These includes are the same as for 
`Condition="'$(Framework)' != 'netcoreapp1.1' AND '$(IndividualAuth)' == 'True'"`

I assume we're expecting them to diverge at some point.